src/components/dapp-staking/my-staking/DynamicAdsArea.vue

```diff
@@ -0,0 +1,109 @@
+    <div class="wrapper-item news-area">
+      <news-area />
+    </div>
+    <div class="wrapper-item wrapper--banners">
+      <div
+        v-for="(t, index) in items"
+        :key="index"
+        class="card"
+        :style="
+          t.gradient
+            ? `background: linear-gradient(180deg, ${t.gradient[0]} 0%, ${t.gradient[1]} 100%)`
+            : ''
+        "
+        @click="goToLink(t.link)"
+      >
+        <div class="wrapper--img">
+          <q-img :src="sourceImg(t.img, index)" class="img--dapp" fit="contain" no-spinner />
+        </div>
+        <div class="card-info">
+          <div class="txt--category">{{ t.category }}</div>
+          <div class="txt--title">
+            {{
+              index === 0 ? $n(truncate(rewardsDeveloper, 0)) + ' ' + nativeTokenSymbol : t.title
+            }}
+          </div>
+          <div class="txt--subtitle">{{ t.subtitle }}</div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+<script lang="ts">
+import { defineComponent, ref, watchEffect } from 'vue';
+import { Struct } from '@polkadot/types';
+import { Perbill } from '@polkadot/types/interfaces';
+import { ethers } from 'ethers';
+import { $api } from 'src/boot/api';
+import { truncate } from 'src/hooks/helper/common';
+import { useNetworkInfo } from 'src/hooks';
+import NewsArea from './components/NewsArea.vue';
+import featuredData from 'src/data/featured_dapp.json';
+
+interface RewardDistributionConfig extends Struct {
+  readonly baseTreasuryPercent: Perbill;
+  readonly baseStakerPercent: Perbill;
+  readonly dappsPercent: Perbill;
+  readonly collatorsPercent: Perbill;
+  readonly adjustablePercent: Perbill;
+  readonly idealDappsStakingTvl: Perbill;
+}
+
+export default defineComponent({
+  components: {
+    NewsArea,
+  },
+  setup() {
+    const items = featuredData;
+    const rewardsDeveloper = ref<number>(0);
+    const { nativeTokenSymbol } = useNetworkInfo();
+
+    const goToLink = (link: string) => {
+      window.open(link, '_blank');
+    };
+
+    const calcDeveloperRewards = async (): Promise<void> => {
+      try {
+        const api = $api!;
+        const [rewardsAllocation, rewardsAmount, blockPerEra] = await Promise.all([
+          api.query.blockReward.rewardDistributionConfigStorage<RewardDistributionConfig>(),
+          api.consts.blockReward.rewardAmount.toString(),
+          Number(api.consts.dappsStaking.blockPerEra.toString()),
+        ]);
+        const rewardsAmountPerEra = Number(ethers.utils.formatEther(rewardsAmount)) * blockPerEra;
+        const numAdjToPercentage = 0.000000001;
+        const dappsPercent = rewardsAllocation.dappsPercent.toNumber() * numAdjToPercentage;
+        rewardsDeveloper.value = rewardsAmountPerEra * dappsPercent;
+      } catch (error) {
+        console.error(error);
+      }
+    };
+
+    const sourceImg = (img: string, index: number) => {
+      if (index === 0) {
+        return require('/src/assets/img/ic_astar_farm.png');
```

Comment:
Shouldn't add image url come from a json?

Reply:
since this image might not be changed, static asset path is hard-coded